packages/gatsby/cache-dir/loader.js

```diff
@@ -516,6 +546,8 @@ export const publicLoader = {
   },
   enqueue: rawPath => instance.prefetch(rawPath),
 
+  getStaticQueryResults: () => instance?.staticQueryDb,
+
```

Comment:
This is not tree shakable and we already have https://github.com/gatsbyjs/gatsby/blob/6858f22fc9a64f71756d5cd5122fab99d12749cf/packages/gatsby/cache-dir/loader.js#L540-L542

that does the same

Reply:
yeah, not sure why I added this now — I thought it didn't work for some reason (or maybe I just didn't notice it).